polkadot-protocol/architecture/polkadot-chain/overview.md

```diff
@@ -1,4 +1,260 @@
 ---
 title: Overview
-description: TODO
----
\ No newline at end of file
+description: Explore the fundamental concepts of the Polkadot protocol, including account addresses, extrinsics, events and runtime upgrades.
+---
+
+# Overview
+
+Since the release of Bitcoin in 2009, blockchain projects increased exponentially to the order of tens of thousands. 
+
+Different projects have different value propositions, suggesting that the future will be multi-chain and that inter-chain communication will be crucial to establish collaborations and leveraging each other strengths.
+
+## Polkadot 1.0
+
+Polkadot 1.0 represents the state of Polkadot as of 2023, coinciding with the release of [Polkadot runtime v1.0.0](https://github.com/paritytech/polkadot/releases/tag/v1.0.0){target=\_blank}. This section will focus on Polkadot 1.0, along with philosophical insights into network resilience and blockspace.
+
+As a Layer-0 blockchain, Polkadot contributes to the multi-chain vision through several key innovations and initiatives, including:
+
+1. **Application-specific Layer-1 blockchains (parachains)** - Polkadot's sharded network allows for parallel transaction processing, with shards that can have unique state transition functions, enabling custom-built L1 chains optimized for specific applications
+
+2. **Shared security and scalability** - L1 chains connected to Polkadot benefit from its [Nominated-Proof-of-Stake (NPoS)](link to the PoS page) system, providing security out-of-the-box without the need to bootstrap their own
```

Comment:
We can probably link the wiki section here: https://wiki.polkadot.network/docs/learn-consensus#nominated-proof-of-stake